RusTaxi/Classes/Screens/Cells/MainController's Cells/Settings Cell/SettingsCell.swift

```diff
@@ -0,0 +1,129 @@
+//
+//  SettingsCell.swift
+//  RusTaxi
+//
+//  Created by Ruslan Prozhivin on 02.09.2018.
+//  Copyright © 2018 App's ID. All rights reserved.
+//
+
+import UIKit
+
+class SettingsCell: UITableViewCell {
```

Comment:
check on iphone 5, it should be as separated cells. One with four labels, second with collectionView